pycbc/events/coinc.py

```diff
@@ -1010,6 +1010,10 @@ def _find_coincs(self, results, ifos):
             [self.ifos[1], self.ifos[0]],
             [[0, -1], [-1, 0]]
         ):
+            if fixed_ifo not in ifos:
```

Comment:
also, since the class init sets `self.ifos = ifos` the use of the same variable name 'ifos' in the coinc call could be confusing ..

Reply:
I've renamed the `ifos` variable to `valid_ifos`, matching how it is used in the parent function. I've also added a docstring for this.